src/commands/synthetics/utils.ts

```diff
@@ -796,6 +807,14 @@ export const renderResults = ({
 
   reporter.runEnd(summary, getAppBaseURL(config))
 
+  reporter.log(`Max parallelization configured: ${orgMaxConcurrencyCap} tests running at the same time\n\n`)
```

Comment:
I think this would be better fitted in the reporters `runEnd` instead of calling `.log` directly from utils. Maybe adding a new `SyntheticsSettings` parameter to it for a non-breaking api change.

We should add a safety condition that `orgMaxConcurrencyCap` is defined and >0, especially since 0 is unlimited I believe and just in case something changes there.
Also may be we don't want to fail the whole run because we didn't manage to retrieve the settings, wdyt? 🤔

nit:
```suggestion
  reporter.log(`Max parallelization configured: ${orgMaxConcurrencyCap} test${orgMaxConcurrencyCap > 1 ? 's' : ''} running at the same time\n\n`)
```